app/organisations.py

```diff
@@ -7,18 +7,22 @@
 
 from app import mongo
 
-organisations = Blueprint("organisations", __name__, template_folder='templates')
+from app.models.user import User
 
+organisations = Blueprint("organisations", __name__, template_folder='templates')
 
 @organisations.route('/', methods=['GET', 'POST'])
 @login_required
 def get_organisations():
     '''
     Display a list of all organisations in table format for admin user
     '''
+
+    # TODO: remove user if not necessary for adding favourite company to a user
+    user = User.find_one_user(session["_user_id"].lower())
     organisations = list(mongo.db.organisations.find())
     return render_template('organisations/list_organisations.html',
-                           organisations=organisations)
+                           organisations=organisations, user=user)
```

Comment:
Since you don't actually need the whole user in the template, could you please just pass in `favourites=user.favourites`